packages/browser-repl/src/components/password-prompt.less

```diff
@@ -0,0 +1,12 @@
+@import '~@leafygreen-ui/palette/dist/ui-colors.less';
+@import './common.less';
+
+.password-prompt input {
```

Comment:
Feels like in the current implementation the prompt is too close to the left side of the shell. Maybe we should add some padding to align it with text above, what do you think?

```suggestion
.password-prompt {
  padding-left: 23px;
}

.password-prompt input {
```

This is how it looks:

<img width="336" alt="image" src="https://user-images.githubusercontent.com/5036933/100246373-5758a280-2f39-11eb-9c16-4d67d5106076.png">

Reply:
Sure, that works for me :)